.github/workflows/tests.yaml

```diff
@@ -195,10 +201,15 @@ jobs:
         uses: actions/checkout@v4
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
+
       - name: Build prod container
         env:
           DOCKER_BUILDKIT: 1
-        run: docker build -f prod.Dockerfile -t civiform:prod --cache-from docker.io/civiform/civiform:latest ./
+        run: bin/build-prod
+
+      - name: Set local prod container tag
+        run: docker tag civiform:latest civiform:prod
+
```

Comment:
Why: `prod-simulator-compose.yml` is looking for that tag. While switching it to the local built version `civiform:latest` I'm not prepared at this moment to make sure there are no issues with it using `latest`.